Source/CombatExtended/Compatibility/WeaponToughnessAutoPatcher.cs

```diff
@@ -18,8 +18,8 @@ static WeaponToughnessAutoPatcher()
             {
                 if (!def.statBases.Any(x => x.stat == CE_StatDefOf.StuffEffectMultiplierToughness || x.stat == CE_StatDefOf.ToughnessRating))
                 {
-                    // Approximate weapon thickness via the bulk of the weapon. Longswords get 2mm, knives get 1mm, spears get about 2.2mm
-                    float weaponThickness = Mathf.Pow(def.statBases?.Find(statMod => statMod.stat.defName == CE_StatDefOf.Bulk.defName)?.value ?? 0f, 1f / 3f);
+                    // Approximate weapon thickness via the bulk of the weapon. Longswords get about 2.83mm, knives get 1mm, spears get about 3.162mm
```

Comment:
In theory, this should be `Mathf.Pow(..., 2f/3f)`, as the 2/3 root gives the average cross-sectional-area from a volume, but since bulk isn't quite ∝ to volume, 1/2 root is probably close enough.  If the values are still too low, it could get bumped slightly though